.github/blueprints/blueprint.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "https://playground.wordpress.net/blueprint-schema.json",
+  "landingPage": "/wp-admin/options-general.php?page=ai-experiments",
+  "preferredVersions": {
+    "php": "8.4",
```

Comment:
A little nervous about this, would feel better dropping to 8.2 unless we KNOW we've tested and things work on 8.4?